src/api/index.js

```diff
@@ -61,5 +61,37 @@ export default ({ config, db }) => {
     }
 	}
 
+	if (config.swagger && config.swagger.enabled) {
```

Comment:
it should be added to `config/default.json` with `swagger.enabled=false` default value. Thanks!